app/src/main/java/org/greenstand/android/TreeTracker/models/user/Users.kt

```diff
@@ -87,6 +93,10 @@ class Users(
         }
     }
 
+    suspend fun checkPlanterExists(identifier: String): Boolean{
+        return dao.checkIfPlanterExists(identifier) >= 1
+    }
```

Comment:
No need for this, just check for a nonnull user from getUserWithIdentifier. There can only ever be one user per identifier/wallet